src/main/java/biblioteka/Calendar.java

```diff
@@ -0,0 +1,354 @@
+package biblioteka;
+import model.Thing;
+import model.User;
+import java.sql.*;
+import java.util.List;
+//https://www.postgresqltutorial.com/postgresql-jdbc/ - fajny tutorial, sa tam tez metody do update'u czy kasowania ale narazie niepotrzebne
+//https://www.postgresqltutorial.com/postgresql-jdbc/query/ - na podstawie tego wyszukiwanie Userow zrobione
+//https://www.postgresqltutorial.com/postgresql-jdbc/insert/ - na podstawie tego dodawanie Userow zrobione
+
+public class Calendar {
+
+    private static String URL = "jdbc:postgresql://localhost:8012/event_base";
+    private static String USER = "postgres";
+    private static String PASSWORD = "password";
+
+    private Connection conn;
+    private Statement stat;
+
+    public Calendar() {
+        try {
+            conn = DriverManager.getConnection(URL, USER, PASSWORD);
+            stat = conn.createStatement();
```

Comment:
We shouldn't share statement object for every of the selects. Please look up Javadoc for the class.

Reply:
I deleted old method createTables(), now I'm doing everything in Calendar contructor., using batch processing. Should it much better from now.